examples/sdk/auth-delegated/web/src/pages/Register.tsx

```diff
@@ -33,9 +43,32 @@ export default function Register(): JSX.Element {
       const challenge = await initRes.json()
       console.log(JSON.stringify(challenge, null, 2))
 
-      // Create the new credential using the challenge
-      const webauthn = new WebAuthn({ rpId: process.env.REACT_APP_DFNS_WEBAUTHN_RPID! })
-      const attestation = await webauthn.create(challenge)
+      let attestation: Fido2Attestation | KeyAttestation
+      // Webauthn flow
+      if (isWebAuthn) {
+        // Create the new webauthn credential using the challenge
+        const webauthn = new WebAuthn({ rpId: process.env.REACT_APP_DFNS_WEBAUTHN_RPID! })
+        attestation = await webauthn.create(challenge)
+      }
+      // Key pair flow
+      else {
+        // Key is generated randomly here
+        // In a production environment they key should be protected 
+        // and loaded securely in the browser
+        const generatedKeyPair = await crypto.subtle.generateKey(
+          { name: 'ECDSA', namedCurve: 'P-256' },
+          true,
+          ['sign', 'verify']
+        )
+        // Here the private key is set as a session variable
+        // key will not exists upon logout
+        setKeyPair(generatedKeyPair)
```

Comment:
I think the key will not even exist upon refresh of the page ? I don't think our app context is persisted in localstorage

Reply:
Yes that's right. It is not the goal of this example (which is about using the sdk) but it highlights that securing the key is not trivial at all.